packages/core/src/virtual-entrypoint.ts

```diff
@@ -147,7 +147,7 @@ export function renderEntrypoint(
 
   return {
     src: entryTemplate(params),
-    watches: [],
+    watches: ['app/**/*'],
```

Comment:
I've tried this locally and it seems to be all we need to get it working 🤷

Reply:
This is another place where we need to be doing things as in d850898eb5c4ea3f58192e3746fcad0ea1180647

Reply:
Instead of hard-coding the string "app".